test/players/human_player_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule HumanPlayerTest do
+  use ExUnit.Case
+  import ExUnit.CaptureIO
+
+  test "cell_selection() returns the correct integer from player's input" do
+    capture_io([input: "1\n"], fn ->
+      assert HumanPlayer.cell_selection(:place_holder) == 1
+    end)
+  end
+
+  test "cell_selection() returns the correct integer from player's input (scenario 2)" do
+    capture_io([input: "1 89%$%}"], fn ->
```

Comment:
What is 89%$% ?

Reply:
lol. just testing what happens if the user enter some gibberish in addition to the cell input

Reply:
maybe I should describe the test exactly how I just explained above.
test "user enter some gibberish in addition to the cell input" do
...